### PR TITLE
Cyb 43 initial code structure

### PR DESCRIPTION
**Pull request by JaegerCryptic**

### CODE REVIEW

**Review by Ganoush69 — APPROVED** [review]
yep, this is code.

File: package.json [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+        "@visx/group": "^3.3.0",
+        "@visx/mock-data": "^3.3.0",
+        "@visx/responsive": "^3.10.2",
+        "@visx/scale": "^3.5.0",
+        "@visx/shape": "^3.5.0",
+        "@visx/text": "^3.3.0",
+        "@visx/tooltip": "^3.3.0",
+        "axios": "^1.6.8",
+        "canvg": "^4.0.1",
+        "countries-state-list": "^1.1.0",
```
Comment (Ganoush69) [inline package.json:51]: what is this

File: src/main.tsx [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+import { ROOT_ROUTE } from './routes/Root'
+import { lightTheme } from './theme/style'
+import { AUTHENTICATED_ROUTES } from './routes/AuthenticatedRoutes'
+import { STARTUP_ROUTE } from './routes/startup/Startup'
+import Notifier from './features/Notifications/Notifier'
+import { LOGOUT_ROUTE } from './routes/Logout'
+import { SearchSummary } from './indexdb/data-source-manager'
+import { useGetDataSourceFuse } from './indexdb/data-source-manager'
+
+const router = createBrowserRouter([
```
Comment (Ganoush69) [inline src/main.tsx:27]: router? im familiar.

**Review by Ganoush69 — APPROVED** [review]
looks good

File: package.json (hunk 1 above)
Comment (JaegerCryptic) [reply package.json:51]: scrap code from earlier we may need later on

File: src/main.tsx (hunk 2 above)
Comment (JaegerCryptic) [reply src/main.tsx:27]: Yup

**Pull request merged**